websites/M/Mixcloud/dist/metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "author": {
-    "name": "Eai",
+    "name": "Eai / LaneSh4d0w",
     "id": "132779412771962880"
```

Comment:
thats not how you add yourself as a contributor see: https://docs.premid.app/en/dev/presence/metadata

Reply:
Whoops.